style.css

```diff
@@ -1003,13 +1003,26 @@ body:not(:-moz-handler-blocked) #single #content > p:first-child:first-letter {
 	margin: 12px 0;
 }
 
+.container .well {
+  border-radius: 4px;
+}
+
+.well {
+  border-radius: 0;
+}
+
+.related-stories .span3:nth-child(4n+1) { margin-left: 0; }
+
 /* Featured Story Styles */
 .featured-story-header { background: #F1F1F1; padding: 1em 20px; margin: 0 -20px 2em -20px; }
```

Comment:
The negative left- and right-hand margins on .featured-story-header need to be moved into a media query for the -xs breakpoint specifically to avoid horizontal scrollbars at -sm+.